scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/dfn.yaml

```diff
@@ -305,6 +305,22 @@ DFN-8-1EP_2x2mm_P0.5mm_EP0.9x1.5mm:
   #include_suffix_in_3dpath: 'False'
   #include_pad_size: 'fp_name_only' # 'both' | 'none' (default)
 
+DFN-8-1EP_2x2mm_P0.5mm_EP0.9x1.6mm:
+  device_type: 'DFN'
+  size_source: 'https://www.st.com/resource/en/datasheet/lm2903.pdf#page=16'
+  body_size_x: 1.85 .. 2.00 .. 2.15
+  body_size_y: 1.85 .. 2.00 .. 2.15
+  lead_width: 0.18 .. 0.25 .. 0.30
+  lead_len: 0.20 .. 0.30 .. 0.50 # only max given in DS.
+                                 # Min and nominal from JEDEC MO-229 V2020D-3
+
+  EP_size_x: 0.9
+  EP_size_y: 1.6
```

Comment:
Datasheet gives tolerance ranges. Please include them here (not used right now but we never know what we might need in future)

Reply:
Done.